sunflower-land/sunflower-land
# [FIX] Island visit UI states

# Description

- add opensea link to inventory (was there in old farm)
- when visiting island:
  - disable place item for chest
  - disable claim button for achievement
  - fix inventory button location

**Inventory icon while visiting**
Before  |  After
---  |  ---
![image](https://user-images.githubusercontent.com/107602352/202866179-c810287f-15b4-4c29-9156-910432481cf3.png)  |  ![image](https://user-images.githubusercontent.com/107602352/202866164-9bbde392-82cc-4c12-bdec-54b9eddbc7cb.png)

**Achievement modal while visiting**
Before  |  After
---  |  ---
![image](https://user-images.githubusercontent.com/107602352/202866194-de974359-ec8f-4731-b451-1654a3e42883.png)  |  ![image](https://user-images.githubusercontent.com/107602352/202866191-9eba348f-9f2f-4875-b895-353495b6d076.png)

**Inventory while visiting**
Before  |  After
---  |  ---
![image](https://user-images.githubusercontent.com/107602352/202866223-c69621bd-169b-4349-ace7-e27e4bc14153.png)  |  ![image](https://user-images.githubusercontent.com/107602352/202866235-8cd860c0-8596-4548-848d-90da07c36c79.png)

## Type of change


# How Has This Been Tested?

- visit a farm, then:
  - check the bottom right of the screen
  - browse their chest
  - browse their achievements

# Checklist:
